src/dbus/common/dbus_message_helper.hpp

```diff
@@ -119,6 +119,10 @@ otbrError DBusMessageEncode(DBusMessageIter *aIter, const TrelInfo &aTrelInfo);
 otbrError DBusMessageExtract(DBusMessageIter *aIter, TrelInfo &aTrelInfo);
 otbrError DBusMessageEncode(DBusMessageIter *aIter, const TrelInfo::TrelPacketCounters &aCounters);
 otbrError DBusMessageExtract(DBusMessageIter *aIter, TrelInfo::TrelPacketCounters &aCounters);
+otbrError DBusMessageEncode(DBusMessageIter *aIter, const BorderAgentInfo &aBorderAgentInfo);
```

Comment:
If you want to add these APIs on DBus, you need to expose them on DBus, and add declaration in src/dbus/server/introspect.xml FYI.

Reply:
Removed as not needed.